notebooks/MosvizExample.ipynb

```diff
@@ -1,66 +1,143 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# MOSViz example notebook"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note: We silence most warnings for now. For debugging, you can comment out the next cell and then restart the kernel to re-enable warnings."
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "from jdaviz.configs.mosviz.helper import MosViz"
+    "import warnings\n",
+    "warnings.simplefilter('ignore')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This starts MOSViz."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "scrolled": false
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "mosviz = MosViz()\n",
+    "from jdaviz.configs.mosviz.helper import MosViz as Mosviz\n",
+    "\n",
+    "mosviz = Mosviz()\n",
     "mosviz.app"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The MOSViz parsers accepts lists of `Spectrum1D`, `SpectralCube`, and `CCDData` for 1D, 2D, and image data respectively. Alternatively, and the method used below, users can provide lists of file paths and MOSViz will internally attempt to parse them as their respective data type."
+    "But before we can use it, we need some data.\n",
+    "\n",
+    "The MOSViz parsers accept lists of `Spectrum1D`, `SpectralCube`, and `CCDData` for 1D, 2D, and image data, respectively. Alternatively, users can also provide lists of file paths and MOSViz will internally attempt to parse them as their respective data types."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "scrolled": false
-   },
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Run this cell if your desired data path is a temporary directory.\n",
+    "\n",
+    "import tempfile\n",
+    "data_dir = tempfile.gettempdir()"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "# *** OR ***\n",
+    "# Enable and run this cell if you have a different data path.\n",
+    "# Replace the value with your real path.\n",
+    "\n",
+    "data_dir = '/path/to/my_mosvis_data_dir'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Run this cell if you wish to download and unzip our example data.\n",
+    "# This requires the \"zipfile\" package.\n",
+    "\n",
+    "from zipfile import ZipFile\n",
```

Comment:
```suggestion
    "from zipfile import ZipFile\n",
    "from astropy.utils.data import download_file\n",
```

Missing an import here

Reply:
I swear I added it when Jesse mentioned it was missing before. The rebase must have reverted my change.